web/src/test/java/com/epam/rft/atsy/web/controllers/rest/CandidateControllerTest.java

```diff
@@ -0,0 +1,284 @@
+package com.epam.rft.atsy.web.controllers.rest;
+
+import com.epam.rft.atsy.service.CandidateService;
+import com.epam.rft.atsy.service.request.FilterRequest;
+import com.epam.rft.atsy.service.request.SearchOptions;
+import com.epam.rft.atsy.service.request.SortingRequest;
+import com.epam.rft.atsy.web.controllers.AbstractControllerTest;
+
+import org.apache.commons.lang3.StringUtils;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.mockito.ArgumentCaptor;
+import org.mockito.InjectMocks;
+import org.mockito.Mock;
+import org.mockito.runners.MockitoJUnitRunner;
+
+import lombok.val;
+
+import static org.hamcrest.MatcherAssert.assertThat;
+import static org.hamcrest.Matchers.equalTo;
+import static org.mockito.BDDMockito.then;
+import static org.mockito.Mockito.verifyNoMoreInteractions;
+import static org.mockito.Mockito.verifyZeroInteractions;
+import static org.springframework.test.web.servlet.request.MockMvcRequestBuilders.get;
+import static org.springframework.test.web.servlet.result.MockMvcResultMatchers.status;
+
+@RunWith(MockitoJUnitRunner.class)
+public class CandidateControllerTest extends AbstractControllerTest {
+
+  private static final String REQUEST_URL = "/secure/candidates";
+
+  private static final String NAME = "name";
+  private static final String ASC = "asc";
+  private static final String CANDIDATE_NAME = "Candidate";
+  private static final String CANDIDATE_PHONE = "+36";
```

Comment:
Please use a bit more realistic phone number and emails, as it is misleading. I believed it is used for an error case.